### ISSUE TITLE
Migration adds fixed value of referencedSchema in references

### ISSUE THREAD

**Issue by siciarek** [phalcon/phalcon-devtools#238]
Migration adds fixed value of referencedSchema in references
it forces developer to use the same table name in dev, test and prod environments

Possible solution:

``` php
protected static $_databaseConfig; // line 53

$referenceDefinition[] = "'referencedSchema' => " . 'self::$_databaseConfig->dbname'; // line 259
```

**Comment by 7thcubic** [comment]
this is a known bug but its not because of migration. Its because of phalcon. I made a fix for 2.0
Dev tools does correctly generate it. Its just the creation part.

https://github.com/7thcubic/cphalcon/commit/2fbfe0a9cbb45bc843960b7511cfc6f1ee955632

**Comment by sergeyklay** [comment]
Fixed in the `3.2.x branch`. Feel free to open a new issue if the problem appears again. Thank you for contributing.
